### ISSUE TITLE
Only video resolution of 320p on Youtube / Embedded Youtube videos are black/not available

### ISSUE THREAD

**Issue by NRubric** [YePpHa/YouTubeCenter#1635]
Since the new Firefox update (37.0.1) every video on Youtube (not embedded Youtube videos on other pages) are only available in 320p when using YouTubeCenter (with Developer Build 497 and Version 2.1.7).

Also embedded Youtube videos outside of Youtube are only showing black where the video should be (even the controls are not appear) with version 2.1.7 but working with Developer Build 497.

Using:
Firefox (37.0.1)
Firefox Addon
YouTubeCenter (Developer Build 497 and Version 2.1.7)

**Comment by LaVey-Charkus-Veros** [comment]
Do you have DASH playback disabled? (YTC Settings >> Player >> General)

**Comment by NRubric** [comment]
Interesting... It seems that disabling DASH causes this resolution problem.

**Comment by LaVey-Charkus-Veros** [comment]
Yes, and it seems this issue started recently, as I had no problems until a few days ago. I also can't download videos higher than 360p. I'm guessing it's a problem with how YouTube is now sending their files.

Anyway, I hate using DASH, as I usually watch very lengthy videos and I like to search through different scenes. With DASH, one has to wait until each segment is buffered.

**Comment by NRubric** [comment]
Yeah... I just read your thread. I also hate using DASH.
